oop-photoenhancer/PhotoEnhancer/PhotoEnhancer/Filters/ContrastParameters.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PhotoEnhancer
+{
+    class ContrastParameters : IParameters
+    {
+        public double Cofficient { get; set; }
```

Comment:
Имя свойства лучше давать без орфографических ошибок
```suggestion
        public double Coefficient { get; set; }
```